java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1364,4 +1364,55 @@ public void zrange_with_different_types_of_keys(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zrangeWithScores(key, query).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zcount(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+
+        assertEquals(
+                3,
+                client
+                        .zcount(key1, InfScoreBound.NEGATIVE_INFINITY, InfScoreBound.POSITIVE_INFINITY)
```

Comment:
If you import the static fields... 
```suggestion
                        .zcount(key1, NEGATIVE_INFINITY, POSITIVE_INFINITY)
```

Reply:
This might clean up your line...

Reply:
I think this should not be imported as we also have InfLexBound